website/documentation/contribute/code/roles/assets/_issue_template.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Membership Request
+about: Request a membership status in the project
```

Comment:
In Kubernetes, there is the https://github.com/kubernetes/org repo, that also automates GitHub org/team membership in a GitOps-style with code reviews using Prow tooling ([Peribolos](https://docs.prow.k8s.io/docs/components/cli-tools/peribolos/)).
We could consider doing something similar to reduce manual and non-auditable actions.

Reply:
I like this idea and concept in general - thanks for sharing. Nevertheless, this doesn't have the highest priority at the moment and I would rather consider it as an enhancement than a prerequisite for the framework described here.

What we can and should do first, is to create a new repo, `gardener/org`, where we store the membership request template (see [comment](https://github.com/gardener/documentation/pull/715#discussion_r2321011349)) and collect enhancements like this one.

Would you agree?

Reply:
> I would rather consider it as an enhancement than a prerequisite for the framework described here.

Definitely!

> What we can and should do first, is to create a new repo, `gardener/org`, where we store the membership request template (see [comment](https://github.com/gardener/documentation/pull/715#discussion_r2321011349)) and collect enhancements like this one.

I like it, sounds like a good starting point for implementing the proposed process 👍